django_usaid_clinicfinder/clinicfinder/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models as djangomodels
+from django.contrib.gis.db import models as gismodels
+from django_hstore import hstore
+
+
+class HStoreModel(djangomodels.Model):
+    objects = hstore.HStoreManager()
+
+    class Meta:
+        abstract = True
+
+
+class Location(gismodels.Model):
+    point = gismodels.PointField()
+    created_at = djangomodels.DateTimeField(auto_now_add=True)
+    updated_at = djangomodels.DateTimeField(auto_now=True)
+
+    # GeoDjango-specific overriding the default manager with a
+    # GeoManager instance.
+    objects = gismodels.GeoManager()
+
+    def __unicode__(self):
+        return "%s" % (self.point)
+
+
+class PointOfInterest(HStoreModel):
```

Comment:
I'm not quite understanding why the ``Location`` and ``PointOfInterest`` models are separate things. Conceptually it feels like they're one entity (e.g. I would be surprised on a map if two points of interest shared a location and changing the location of one also moved the other). Mostly just raising this for consideration.